src/video_core/buffer_cache/buffer_cache.h: Missing include?

I'm building Yuzu on Ubuntu 19.04 with GCC 9.1.0. It gives me this error:

> /home/cleverking/yuzu/src/./video_core/buffer_cache/buffer_cache.h:267:34: error: «make_iterator_range» is not a member of «boost»
  267 |         for (auto& pair : boost::make_iterator_range(mapped_addresses.equal_range(interval))) {
      |     

It looks like there's a missing include. When I add #include <boost/range.hpp>, Yuzu compiles normally.
[build_log.txt](https://github.com/yuzu-emu/yuzu/files/3558846/build_log.txt)

--- a/src/video_core/buffer_cache/buffer_cache.h
+++ b/src/video_core/buffer_cache/buffer_cache.h
@@ -12,6 +12,8 @@
 #include <utility>
 #include <vector>
 
+#include <boost/range/iterator_range.hpp>
+
 #include "common/alignment.h"
 #include "common/common_types.h"
 #include "core/core.h"

--- a/src/video_core/buffer_cache/buffer_cache.h
+++ b/src/video_core/buffer_cache/buffer_cache.h
@@ -12,6 +12,8 @@
 #include <utility>
 #include <vector>
 
+#include <boost/icl/interval_map.hpp>
+#include <boost/icl/interval_set.hpp>
 #include <boost/range/iterator_range.hpp>
 
 #include "common/alignment.h"

